lib/ruby_lsp/ruby_lsp_rails/hover.rb

```diff
@@ -145,6 +145,7 @@ def handle_association(node)
 
         return unless result
 
+        generate_column_content(result[:name])
```

Comment:
Adding this as is here works, but it means that any hover on associations needs two round trips to the server to pull information.

Instead of doing that, could we include the required information as part of the `association_target` response and then extract what's needed to generate the hover?

That way we get all of the data in a single request.